breakdown.go

```diff
@@ -129,8 +126,8 @@ func newBreakdownMetricsMap() *breakdownMetricsMap {
 }
 
 type breakdownMetricsMapEntry struct {
-	breakdownTiming
 	breakdownMetricsKey
+	breakdownTiming
```

Comment:
Moved this here to align the struct better and reduce its size, I thought it worth pursuing since it may have up to 1000 entries in memory:

```
struct with 80 pointer bytes could be 56
```

Doing so removes 24 bytes of padding per entry, which can amount up to 24Kbytes of savings, not huge but less memory usage nevertheless.

Reply:
nice :)